kinto/__main__.py

```diff
@@ -52,13 +54,17 @@ def main(args=None):
                               default=False)
     parser_start.set_defaults(which='start')
 
-    args = vars(parser.parse_args())
+    try:
+        args = vars(parser.parse_args(args))
+    except SystemExit:  # pragma: no cover for python2
+        return 10
```

Comment:
I didn't understand the reason that this got added in the first place, so I'm not qualified to comment on that.. isn't this for the purposes of testing or something?